.rspec

```diff
@@ -0,0 +1,3 @@
+--color
+--format progress
+--require rails_helper
```

Comment:
Requiring `rails_helper` here so we don't need to add `require 'rails_helper'` to the top of every spec file.